File: src/main/java/gay/skitbet/jackiro/command/impl/PingCommand.java
package gay.skitbet.jackiro.command.impl;

import gay.skitbet.jackiro.command.Command;
import gay.skitbet.jackiro.command.CommandContext;
import net.dv8tion.jda.api.interactions.commands.build.OptionData;

import java.util.List;

public class PingCommand extends Command {
    public PingCommand() {
        super("ping", "PONG PONG PONG");
    }

    @Override
    public void execute(CommandContext context) {
        context.reply("PONGG!");
    }

    @Override
    public List<OptionData> addOptions(List<OptionData> options) {
        return options;
    }
}

File: src/main/kotlin/gay/skitbet/jackiro/command/impl/PingCommand.kt
package gay.skitbet.jackiro.command.impl

import gay.skitbet.jackiro.command.Command
import gay.skitbet.jackiro.command.CommandContext
import net.dv8tion.jda.api.interactions.commands.build.OptionData

class PingCommand : Command("ping", "PONG PONG PONG", null, true) {
    override fun execute(context: CommandContext) {
    }

    override fun addOptions(options: MutableList<OptionData?>): MutableList<OptionData?> {
        return options
    }

}
